TODO:
src/requestobject.c uses ap_requires() which has been removed in r368027.
https://svn.apache.org/viewvc?view=rev&revision=368027

--- src/include/mod_python.h.in
+++ src/include/mod_python.h.in
@@ -72,6 +72,11 @@
 #endif
 #endif
 
+#if !AP_MODULE_MAGIC_AT_LEAST(20081201,0)
+#define ap_unixd_config unixd_config
+#define ap_unixd_set_global_mutex_perms unixd_set_global_mutex_perms
+#endif
+
 /* Python headers */
 /* this gets rid of some compile warnings */
 #if defined(_POSIX_THREADS)
--- src/mod_python.c
+++ src/mod_python.c
@@ -555,14 +555,14 @@
         else {
 
             /*XXX As of httpd 2.0.4, the below should be just
-              a call to unixd_set_global_mutex_perms(mutex[n]); and
+              a call to ap_unixd_set_global_mutex_perms(mutex[n]); and
               nothing else... For now, while 2.0.48 isn't commonplace yet,
               this ugly code should be here */
 
 #if !defined(OS2) && !defined(WIN32) && !defined(BEOS) && !defined(NETWARE)
             if  (!geteuid()) {
-                chown(fname, unixd_config.user_id, -1);
-                unixd_set_global_mutex_perms(mutex[n]);
+                chown(fname, ap_unixd_config.user_id, -1);
+                ap_unixd_set_global_mutex_perms(mutex[n]);
             }
 #endif
         }
